### PR TITLE
fix compiling errors for 404-server

### PR DESCRIPTION
**Pull request by dixudx**
Fix compiling errors that brought by new packages added in #2376.

### CODE REVIEW

File: 404-server/Makefile [hunk 1]
```diff
@@ -27,12 +27,12 @@ GOLANG_VERSION=1.7
 TEMP_DIR:=$(shell mktemp -d)
 
 server: server.go
-	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) GOARM=6 go build -a -installsuffix cgo -ldflags '-w -s' -o server ./server.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) GOARM=6 go build -a -installsuffix cgo -ldflags '-w -s' -o server ./server.go ./metrics.go
```
Comment (luxas) [inline 404-server/Makefile]: nit: I guess `.` or nothing just works here as well?
Not optimal to enumerate all files here if/as the source grows in size

**Review by luxas — COMMENTED** [review]
LGTM, one nit

File: 404-server/Makefile (hunk 1 above)
Comment (dixudx) [inline 404-server/Makefile]: @luxas Yeah, you're right. Updated. PTAL. Thanks.

**Comment by luxas** [comment]
/lgtm

cc @bprashanth 
Seems like Travis is broken. Do you know of any fix?

**Comment by spiffxp** [comment]
/reopen
trying to retrigger travis

**Pull request merged**